ansible-st2/roles/arteria_node/tasks/arteria_runfolder.yml

```diff
@@ -0,0 +1,82 @@
+---
+
+#TODO Ensure all this is versioned!
+
+- name: get arteria-core from github for runfolder
```

Comment:
Have added commits that moves the tags higher up, and disables iptables via a variable instead. I guess the dependency should be fixed in the services first.